### DISCUSSION TITLE
Activity Bar error

### DISCUSSION THREAD

**Discussion by azamat684** [community/community#113992] [New to GitHub]
### Hi everyone,

I have an issue that when I pushed codes from my local to GitHub, it doesn't show on my activity bar but I changed smth on repo it shows correctly on activity bar. What should I do for solve this? ( I wrote email and username on my terminal correctly and all codes go right destination.)

**Comment by samus-aran** [comment]
Welcome to the GitHub Community, @azamat684 , we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!

**Comment by jfullstackdev** [comment]
you might be pushing on a different branch, sure you pushed or merge that in the default branch ? I'm assuming you are using GitHub Page, please provide more info so we can help

**Reply by azamat684** [reply]
hi. First of all SORRY IF I MAKE MISTAKE WHILE I TEXTING YOU IN ENGLISH )). I faced this kind of problem even I don't have a branch on repo. However when I change something on online page github, It works correctly. I mean it shows on active bar. When I push codes from my local to github, nothing changes and shows on active bar.
